robo-components/BootstrapTrait.php

```
@@ -187,18 +189,27 @@ protected function prepareGithubRepository(string $project_name, string $organiz
    *   The project name.
    * @param string $project_machine_name
    *   The project machine name in GH slug.
-   * @param string $organization
-   *   The GitHub/Pantheon organization.
    */
-  protected function createPantheonProject(string $terminus_token, string $project_name, string $project_machine_name, string $organization) {
+  public function createPantheonProject(string $terminus_token, string $project_name, string $project_machine_name) {
     $result = $this->taskExec("terminus auth:login --machine-token=\"$terminus_token\"")
       ->run()
       ->getExitCode();
     if ($result !== 0) {
       throw new \Exception("Failed to login to Terminus.");
     }
 
-    $result = $this->taskExec("terminus site:create $project_machine_name \"$project_name\" \"bde48795-b16d-443f-af01-8b1790caa1af\" --org=\"$organization\"")
+    $organizations = $this->taskExec("terminus org:list --format=json")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+    $organizations = json_decode($organizations, TRUE);
+
+    // Prompt the user to select an organization.
+    $io = new ConsoleIO($this->input(), $this->output());
+    $organization_choices = array_combine(array_column($organizations, 'id'), array_column($organizations, 'label'));
```

Maybe if there's a single org don't ask?

---

Yep, and when no org, we can throw an exception.